### DISCUSSION TITLE
As a CEA user, do you want to see CEA interface (plug-in) for Grasshopper?

### DISCUSSION THREAD

**Discussion by ShiZhongming** [architecture-building-systems/CityEnergyAnalyst#3326] [Ideas]
It is coming back to the table if CEA should re-start its interface for Rhino/Grasshopper. I am initiating this post to discuss with the CEA Team and all CEAers to check if you are willing to have the CEA plug-in for Grasshopper.

The screenshot below presents the PV Component of CEA for Grasshopper we used to have in 2018.

![three+technologies](https://github.com/architecture-building-systems/CityEnergyAnalyst/assets/16640327/78388a7e-31f5-4a04-b390-db35fd9a18a1)

**Comment by anyco2018** [comment]
Yes, that would be useful.

**Reply by norafankz** [reply]
That would very useful, I am an Urban Computational Designer and work daily on RH/GH, would be amazing to have a plugin for gh, so we we can directly input the 3d content!
